code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -91,23 +91,23 @@
 		/obj/item/autosurgeon/organ/syndicate/meson_eyes, // 0TC
 		/obj/item/encryptionkey/syndicate) // 2TC
 
-/// 175TC
+/// 185TC
 /obj/item/storage/box/syndie_kit/bundle/payday
 	name = "Heist Bundle"
 	desc = "Alright guys, today we're performing a heist on a space station owned by a greedy corporation. Drain the vault of all its worth so we can get that pay dirt!11"
```

Comment:
"dirt!11" do you think this is a typo on the description?

Reply:
I counted the items, adds up to the amount. I guess I could be wrong but I like triple checked.

Reply:
Learning a new meme with each day

Reply:
"!11" is not a typo, it was copied from the syndicate equipment wiki page, as descriptions come from there first